docs/administering/install-troubleshooting.md

```diff
@@ -129,6 +129,18 @@ sudo service sandstorm restart
 
 This should resolve your problems. If not, please get in touch.
 
+## Are grains not starting on Ubuntu 24.04 or later?
+
+If your wildcard DNS is configured correctly, you should see the app icons correctly on the apps tab. If
+grains are not starting, and you are on Ubuntu 24.04 or later, AppArmor may be restricting unpriviliged
+user namespaces. You can run the following commands to correct this:
+
+```bash
+sudo echo "kernel.apparmor_restrict_unprivileged_userns = 0" > /etc/sysctl.d/99-userns.conf
```

Comment:
Did you actually run this successfully? I think the `sudo` powers wear off when you hit the `>` token:

```
sudo echo "test" > /etc/test-test-test
bash: /etc/test-test-test: Permission denied
```

Reply:
Also how comfortable are you with `99-userns` being unique enough? That line (once permissions are fixed) would overwrite anything the user might have had there if they happened to have the same filename.

Reply:
I can do some testing with our VPS demo. Maybe we should change it to sandstorm-userns.conf or something so it's extremely unique?